plugins/hls-eval-plugin/src/Ide/Plugin/Eval/Handlers.hs

```diff
@@ -84,6 +85,7 @@ import qualified Development.IDE.GHC.Compat.Core              as SrcLoc (unLoc)
 import           Development.IDE.Types.HscEnvEq               (HscEnvEq (hscEnv))
 import qualified GHC.LanguageExtensions.Type                  as LangExt (Extension (..))
 
+import           Data.List.Extra                              (unsnoc)
```

Comment:
`unsnoc` is imported from `Data.List.Extra` instead of `Data.List` because `Data.List` does not have `unsnoc` for `base < 4.19.0.0` which causes CI failures.